src/components/pages/ManageDomain/ManageDomain.tsx

```diff
@@ -3,14 +3,95 @@ import DomainSettings from '@src/components/forms/DomainSettings/DomainSettings'
 import { usePrimaryName } from '@src/hooks/usePrimaryName';
 import { useGlobalState, useModalState } from '@src/state';
 import { useTransactionState } from '@src/state/contexts/TransactionState';
+import eventEmitter from '@src/utils/events';
+import { useQuery } from '@tanstack/react-query';
 import { Star } from 'lucide-react';
-import { useEffect } from 'react';
+import { ReactNode, useEffect, useState } from 'react';
 import { useNavigate, useParams } from 'react-router-dom';
 
 import { decodeDomainToASCII } from '../../../utils';
 import { HamburgerOutlineIcon } from '../../icons';
 import './styles.css';
 
+function AntLogoIcon({
+  id,
+  className,
+  icon = (
+    <HamburgerOutlineIcon
+      width={'20px'}
+      height={'20px'}
+      fill="var(--text-white)"
+    />
+  ),
+}: {
+  id?: string;
+  className?: string;
+  icon?: ReactNode;
+}) {
+  const [{ gateway }] = useGlobalState();
+
+  const { data: logoImage } = useQuery({
+    queryKey: ['ant-logo', id, gateway],
+    queryFn: async () => {
+      try {
+        if (!id) return;
+        const imageRes = await fetch(`https://${gateway}/${id}`);
+        const buffer = await imageRes.arrayBuffer();
+        const arr = new Uint8Array(buffer).subarray(0, 4);
+        const header = Array.from(arr)
+          .map((byte) => byte.toString(16))
+          .join('');
+
+        const imageHeaders = {
+          '89504e47': 'image/png', // PNG
+          '47494638': 'image/gif', // GIF
+          ffd8ffe0: 'image/jpeg', // JPEG
+          ffd8ffe1: 'image/jpeg', // JPEG
+          ffd8ffe2: 'image/jpeg', // JPEG
+          ffd8ffe3: 'image/jpeg', // JPEG
+          ffd8ffe8: 'image/jpeg', // JPEG
```

Comment:
add webp and svg

Reply:
[20a0a51](https://github.com/ar-io/arns-react/pull/598/commits/20a0a5139141c5530b4d2a73e7ce179daa79bfe3)